pkg/auth/auth.go

```diff
@@ -0,0 +1,162 @@
+package auth
+
+import (
+	sha "crypto/sha512"
```

Comment:
@acud what is the best library to generate access tokens? should we leave `sha512` or is there a better fit?

Reply:
not sure if we need a library for that... might be that using `urandom` might be enough. or guid. not sure. would be happy for input from the JS team here

Reply:
From the perspective of the API user it does not matter how it is created, this is just an opaque string that needs to be kept in memory so that the requests are authorized. It helps if it is base64 encoded so that it can be easily copy-pasted and used from CLI tools without escaping.

Reply:
swapped for a simpler solution, please review.